lib/Form/FormInput.js

```diff
@@ -50,26 +54,35 @@ class FormInput extends Component {
       value,
       fullWidth,
       paddingSmall,
+      hasError,
+      errorMessage,
       ...rest
     } = this.props;
     const inputValue = this.state.value;
 
     const classNames = cx(`form__input ${className}`, {
       'form__input--noborder': noBorder,
       'form__input--full-width': fullWidth,
-      'form__input--padding-small': paddingSmall
+      'form__input--padding-small': paddingSmall,
+      'form__input--has-error': hasError,
+      'form__input--display-error': hasError && errorMessage
     });
 
     return (
-      <input
-        type={type}
-        value={inputValue}
-        onChange={this.handleOnChange}
-        className={classNames}
-        placeholder={placeholder}
-        autoFocus={this.props.focusOnMount} // eslint-disable-line jsx-a11y/no-autofocus
-        {...rest}
-      />
+      <Fragment>
+        <input
+          type={type}
+          value={inputValue}
+          onChange={this.handleOnChange}
+          className={classNames}
+          placeholder={placeholder}
+          autoFocus={this.props.focusOnMount} // eslint-disable-line jsx-a11y/no-autofocus
+          {...rest}
+        />
+        {hasError && errorMessage && (
+          <span className="form-input__error-message">{errorMessage}</span>
```

Comment:
When this validation message appears, users with screen readers won't be notified, so they won't know there is an error to be fixed! To make this happen, you can add the attribute `role="alert"` which will screen read whatever is inside the span when it appears.